tests/test_spyre_online_multi.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from spyre_util import RemoteOpenAIServer
+
+from tests.spyre_util import get_spyre_backend_list, get_spyre_model_list
+
+
+@pytest.mark.parametrize("model", get_spyre_model_list())
+# (prompt_length/new_tokens/batch_size)
+@pytest.mark.parametrize("warmup_shape", [[
+    (64, 20, 4),
+]])
+@pytest.mark.parametrize("backend", get_spyre_backend_list())
+@pytest.mark.parametrize("tensor_parallel_size", [2])
+@pytest.mark.parametrize("vllm_version", ["V0", "V1"])
+def test_openai_serving(model, warmup_shape, backend, tensor_parallel_size,
```

Comment:
Nice!

Any thoughts on moving this over to `test_spyre_tensor_parallel.py`? I kinda think we should try to start organizing these files a bit instead of creating more of them. I can also move my basic online tests over to `test_spyre_basic.py`

Reply:
I think that's cleaner, I just wonder if in the future you could see a situation where online tests would have to be refactored/separated from offline. Up to you

Reply:
Also, online tests are passing/exiting properly without --forked

Reply:
>I think that's cleaner, I just wonder if in the future you could see a situation where online tests would have to be refactored/separated from offline. Up to you

Yeah... that could end up being the case as well 🤷 
We can leave test organization until later